en/micro-integrator/docs/references/connectors/file-connector/file-connector-example.md

```diff
@@ -0,0 +1,138 @@
+            <fileconnector.create>
+                <source>{$ctx:source}</source>
+                <inputContent>{$ctx:inputContent}</inputContent>
+            </fileconnector.create>
+            <respond/>
+        </inSequence>
+        <outSequence/>
+        <faultSequence/>
+    </resource>
+    <resource methods="POST" uri-template="/read">
+        <inSequence>
+            <property expression="json-eval($.source)" name="source" scope="default" type="STRING"/>
+            <fileconnector.isFileExist>
+                <source>{$ctx:source}</source>
+            </fileconnector.isFileExist>
+            <property expression="json-eval($.fileExist)" name="response" scope="default" type="STRING"/>
+            <log level="custom">
+                <property expression="get-property('response')" name="responselog"/>
+            </log>
+            <switch source="get-property('response')">
+                <case regex="true">
+                    <fileconnector.read>
+                        <source>{$ctx:source}</source>
+                    </fileconnector.read>
+                    <respond/>
+                </case>
+                <default>
+                    <log>
+                        <property name="notext" value="&quot;File does not exist&quot;"/>
+                    </log>
+                </default>
+            </switch>
+            <drop/>
+        </inSequence>
+        <outSequence/>
+        <faultSequence/>
+    </resource>
+</api>
+
+```
+
+4. Right-click on the Composite Application Project and click on **Export Project Artifacts and Run**. Export tha carbon application to your desired location. 
+
+## Deployment
+### Micro Integrator
+
+You can copy the carbon application to the <PRODUCT-HOME>/repository/deployment/server/carbonapps folder and start the server. 
+Micro Integrator will be started and the composite application will be deployed. You can further refer to the application deployed through the CLI tool. Make sure you first export the PATH as below.
+
+    ```
+    $ export PATH=/path/to/mi/cli/directory/bin:$PATH\
+    ```
+
+1. Login.
+
+    ```
+    ./mi remote login
+    ```
+
+2. Provide default credentials admin for both username and password.
+3. In order to view the APIs deployed, execute the following command.
+
+    ```
+    ./mi api show
+    ```
+
+### wso2-ei-6.6.0
+
+1. You can copy the carbon application to the <PRODUCT-HOME>/repository/deployment/server/carbonapps folder and start the server. 
+
+2. WSO2 EI server will be started and you can login to the Management Console https://localhost:9443/carbon/ URL. Provide login credentials. 
+
+3. Under API section it can be seen that the API is deployed. 
+
+## Testing
+
+### File Create Operation
+
+1. Create a file called data.json with the following payload. 
+```
+{
+    "source":"/Users/IsuruUyanage/Desktop/RnD/2020/Q1/ConnectorVerification/create.txt",
+    "inputContent": "This is a test file"
+}
+```
+2. Invoke the API as below. 
```

Comment:
Let's include the response/expected result. Same comment applies for the next section as well.

Reply:
Done